models/Trail.js

```diff
@@ -0,0 +1,42 @@
+const { Model, DataTypes } = require('sequelize');
+const sequelize = require('../config/connection');
+
+class Trail extends Model {}
+
+Trail.init(
+  {
+    //id, name, location, distance, elevation, difficulty
+    id: {
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        primaryKey: true,
+        autoIncrement: true,
+    },
```

Comment:
Can you add a "name" parameter to the Trail model (type: string, allowNull: false)